core/benches/validation.rs

```diff
@@ -224,7 +198,6 @@ criterion_group!(
 );
 criterion_group!(
     blocks,
-    chain_blocks,
```

Comment:
Why removed?

Reply:
Already removed on dev, conversation does not apply anymore.